docs/Configuration.md

```diff
@@ -714,6 +714,77 @@ url: the URL of the GBFS feed (do not include the gbfs.json at the end) *
 ```
 \* For a list of known GBFS feeds see the [list of known GBFS feeds](https://github.com/NABSA/gbfs/blob/master/systems.csv)
 
+#### TNC Configuration
+
+It is possible to add either Uber or Lyft as a TNC updater. OTP provides helper methods for making authenticated requests to each provider's API and also for verifying that TNC service exists when making certain routing requests that use a TNC for part or all of the trip.
+
+For each TNC updater, you have to manually determine what the ride type ids are for wheelchair-accessible services in the area that OTP will be planning trips in.
+
+##### Uber
+
+- Add one entry in the `updater` field of `router-config.json` in the format:
+
+```JSON
+{
+    "type": "transportation-network-company-updater",
+    "sourceType": "uber",
+    "serverToken": "YOUR-SECRET-TOKEN",
+    "wheelChairAccessibleRideType": "REGION-SPECIFIC-ID"
+}
+```
+
+##### Lyft
+
+- Add one entry in the `updater` field of `router-config.json` in the format:
+
+```JSON
+{
+    "type": "transportation-network-company-updater",
+    "sourceType": "lyft",
+    "clientId": "YOUR-CLIENT-ID-IF-LYFT-ISNT-BEING-ALL-WALLED-GARDEN-WITH-YOU",
+    "clientSecret": "YOUR-CLIENT-SECRET-IF-LYFT-ISNT-BEING-ALL-WALLED-GARDEN-WITH-YOU"
+}
+```
+
+#### Car Rental Configuration
+
+Currently, it is possible to add an updater for either car2go or the now defunct ReachNow. Both of these updaters read in data from a url or file about the vehicle positions and the areas where the cars are allowed to be dropped off inside of. The data format of the vehicle positions slightly differs between the each updater, but they're both a list of JSON objects. The regions data must be valid GeoJSON of Polygons or MultiPolygons in either a Feature or a Feature collection.
+
```

Comment:
Since ReachNow is dead, should we just remove the updater (or at very least deprecate it)? My inclination is to remove since it is now essentially impossible to use this updater in any real-world context.